hw/top_darjeeling/templates/chiplevel.sv.tpl

```diff
@@ -834,12 +834,27 @@ module chip_${top["name"]}_${target["name"]} #(
     .tl_d2h_i   (dmi_d2h)
   );
 
+  // TODO: Resolve this and wire it up.
```

Comment:
We should create issues to track these TODOs (unless they get resolved by a later commit in this PR).

Reply:
Shall do, thanks.

Reply:
Tracking in issues as requested.